projects/angular-calendar/schematics/ng-add/schema.json

```diff
@@ -24,10 +24,32 @@
         ]
       }
     },
+    "standalone": {
+      "type": "boolean",
+      "default": false,
+      "description": "Install to a standalone component (default) or NgModule",
+      "x-prompt": {
+        "message": "Install to a standalone component or NgModule?",
+        "type": "list",
+        "items": [
+          {
+            "value": true,
+            "label": "Standalone component (recommended)"
+          },
+          {
+            "value": false,
+            "label": "NgModule"
+          }
+        ]
+      }
+    },
     "module": {
```

Comment:
Rename this `module` option to `installToPath` (rename any other references in the schema / schematics files etc)